Runtime/LODGenerator.cs

```diff
@@ -166,11 +166,16 @@ public static LODGroup GenerateLODs(GameObject gameObject, LODLevel[] levels, bo
                 {
                     var meshRenderers = (from renderer in originalLevelRenderers
                                          where renderer.enabled && renderer as MeshRenderer != null
+                                        && renderer.transform.GetComponent<MeshFilter>() != null
+                                        && renderer.transform.GetComponent<MeshFilter>().sharedMesh != null
```

Comment:
I would prefer if the component was only retrieved once.
Perhaps creating a helper method for this?